dags/ltv.py

```diff
@@ -118,4 +118,15 @@
     time_partitioning={"type": "DAY", "field": "submission_date"},
 )
 
-ltv_daily >> ltv_revenue_join
+ltv_normalized_view=BigQueryOperator(
+    task_id='ltv_normalized_view',
+    sql=response.read().decode('utf-8'),
+    query_params=[{"name": "submission_date", "parameterType": {"type": "DATE"}, "parameterValue": {"value": "{{ ds }}"}}],
+    destination_dataset_table='moz-fx-data-shared-prod.revenue_derived.client_ltv_v1${{ ds_nodash }}',
```

Comment:
Two things here:
1. This dataset does not currently exist
2. This connection id may not have write access to it

cc @jasonthomas, can you help me out with those?